docs/usage.rst

```diff
@@ -138,6 +138,12 @@ value more suitable for you. This value, which defaults to `None`, will be passe
 `Django's bulk_create method <https://docs.djangoproject.com/en/stable/ref/models/querysets/#bulk-create>`_
 as the `batch_size` parameter.
 
+If you want to change how errors are handled you can create your own error handler
+and set ``MAILER_ERROR_HANDLER`` as a string for import (default is
+``"mailer.engine.handle_backend_exception"``). Your handler should return a connection
+(could be ``None`` causing a new connection to be created) and an action_taken (one
+of ``sent`` or ``deferred``).
```

Comment:
Thanks for the two layers of quotes above. Can we have it for `"sent"` and `"deferred"` as well, for consistency? Thanks! :pray: